.github/workflows/format.yaml

```diff
@@ -11,13 +11,17 @@ jobs:
 
     steps:
 
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 2
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: coursier-cache-action
-        uses: coursier/cache-action@v6
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+          cache: sbt
```

Comment:
setup-java@v4 has support for sbt caching; sbt uses coursier. So, the coursier/cache-action is no longer necessary.